packages/@react-spectrum/s2/stories/Button.stories.tsx

```diff
@@ -48,3 +50,52 @@ export const Example: Story = {
     );
   }
 };
+
+
+export const PendingButton = {
```

Comment:
Unfortunately that won't trigger the announcement since you really need to have focus on the button. So in terms of actually experiencing the behavior we've built, it's better to have the story a bit more interactive than usual I think.
I'll exclude this story from the docs though.